client/app/components/avatar.tsx

```diff
@@ -2,21 +2,21 @@ import BoringAvatar from "boring-avatars";
 import clsx from "clsx";
 
 interface Props {
-	name: string;
-	size?: number;
-	className?: string;
+  name: string;
+  size?: number;
+  className?: string;
 }
 
 const colors = ["#ffe12e", "#4d8c3a", "#0060ff", "#ff7d10", "#4e412b"];
 
-const BA = BoringAvatar.default;
+const BA = BoringAvatar;
```

Comment:
I'm not sure you tried to run the project; as doing this causes compilation error

Reply:
There's no `default` property

Reply:
Did you run project?

Reply:
Yeah. I couldn't run it prior to removing the `default` property because there is no `default` property. TS error. Check the [docs](https://github.com/boringdesigners/boring-avatars?tab=readme-ov-file#usage)